conans/client/loader.py

```diff
@@ -19,8 +18,9 @@ def _load_info_file(current_path, conanfile, output, error):
         return
     info_file_path = os.path.join(current_path, BUILD_INFO)
     try:
-        deps_info = DepsCppInfo.loads(load(info_file_path))
-        conanfile.deps_cpp_info = deps_info
+        deps_cpp_info, deps_user_info = TXTGenerator.loads(load(info_file_path))
+        conanfile.deps_cpp_info = deps_cpp_info
+        conanfile.deps_user_info = deps_user_info
```

Comment:
If https://github.com/conan-io/conan/pull/1481 is accepted, implement the same approach for it.